srv/ChangeCameras.srv

```diff
@@ -1,5 +1,6 @@
 bool primary # true if requesting streams for primary laptop
-CameraCmd[4] camera_cmds
+CameraCmd camera_cmds
```

Comment:
Can we drop the s on this variable since it's not plural

Reply:
yes sure